cmd/juju/charmhub/download.go

```diff
@@ -37,6 +37,10 @@ by a specified name. Downloading for a specific base can be done via
 --base. --base can be specified using the OS name and the version of
 the OS, separated by @. For example, --base ubuntu@22.04.
 
+By default, the latest revision in the stable channel will be
```

Comment:
s/stable/default/

Stable is ambiguous here as it's a risk and used to indicate the stable risk of the default channel in call to charm hub. The default channel is charm author defined with a fall back to latest.